### DISCUSSION TITLE
Compatibility with WooCommerce Blocks

### DISCUSSION THREAD

**Discussion by webaxones** [woocommerce/woocommerce#39002] [WooCommerce New Product Editor] [answered]
I can understand that WooCommerce Product Block Editor may not be compatible with plugins like WooCommerce Product Add-ons yet, but why is it also not compatible with WooCommerce Blocks? I thought WC Blocks was the equivalent of Gutenberg with code backports, so I would have expected to see this feature first in WC Blocks and then in the WC core. However, I certainly didn't expect to see this feature incompatible with this extension. :)

**Comment by louwie17** [comment, marked as answer]
Hi @webaxones, thanks for the question.
When first releasing the feature flag we enabled the declare dependency logic ( similar to HPOS ), which would declare any WooCommerce aware plugin incompatible by default, unless it had explicitly declared compatibility within the code. We haven't declared compatibility within the WooCommerce Blocks plugin yet, although it is compatible.
Instead, for the upcoming release 7.9, we disabled the 'dependency logic', so you can enable the feature flag even if you have multiple WooCommerce plugins enabled.

**Reply by webaxones** [reply]
Thanks!
